pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -919,61 +754,6 @@ spec:
 			name:  "sa-step",
 			cmd:   "/mycmd",
 		}}),
-	}, {
-		name:    "params",
-		taskRun: taskRunSubstitution,
```

Comment:
Why is this test removed?
`templatedTask` is still around and not tested (one line still includes a reference to a pipeline resource).
The other substituions are still valid tests value, various variables defined in args and elsewhere

Reply:
ty for the catch, restored and replaced the resources parts